### PR TITLE
Update load_balance.md

### PR DESCRIPTION
**Pull request by arthuo**
Overwrite with new ones_zh

### CODE REVIEW

File: site/zh-CN/userGuide/load_balance.md [hunk 1]
```diff
@@ -4,19 +4,19 @@ related_key: Load balance
...
-If automatic load balance is disabled, you can still balance the load manually.
+Milvus 默认支持自动负载均衡。你可以[配置](configure-docker.md)  Milvus 以启用或禁用[自动负载均衡](configure_querycoord.md#queryCoordautoBalance)。通过指定 [`queryCoord.balanceIntervalSeconds`](configure_querycoord.md#queryCoordbalanceIntervalSeconds), [`queryCoord.overloadedMemoryThresholdPercentage`](configure_querycoord.md#queryCoordoverloadedMemoryThresholdPercentage) 和 [`queryCoord.memoryUsageMaxDifferencePercentage`](configure_querycoord.md#queryCoordmemoryUsageMaxDifferencePercentage)，你可以更改触发自动负载均衡的阈值。
 
-## Check segment information
+如果禁用自动负载均衡，你仍然可以手动平衡负载。
 
-Get the `segmentID` of the sealed segment that you expect to transfer and the `nodeID` of the query node that you expect to transfer the segment to.
+## 查看 segment 信息
+
+获取输入 collection 的封装段的 `segmentID` 和查询节点的 `nodeID` 。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:19]: 封装段-->sealed segment
查询节点-->query node

File: site/zh-CN/userGuide/load_balance.md [hunk 2]
```diff
@@ -50,66 +50,66 @@ show query_segment -c book
...
+            <td>用于查看 segment 信息的 collection 名称。</td>
         </tr>
     </tbody>
 </table>
 
-## Transfer segment
+## 迁移 segment
 
-Transfer the sealed segment(s) with the `segmentID` and the `nodeID` of the current query node and new query node(s).
+使用当前查询节点和新查询节点的 `segmentID` 和 `nodeID` 迁移封装 segment 。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:112]: 查询节点-->query node

File: site/zh-CN/userGuide/load_balance.md [hunk 3]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
-		<th>Description</th>
+		<th>参数</th>
+		<th>说明</th>
 	</tr>
 	</thead>
 	<tbody>
 	<tr>
 		<td><code>src_node_id</code></td>
-		<td>ID of the query node you want to transfer segment(s) from.</td>
+		<td>源 segment 查询节点的 ID 。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:158]: 查询节点-->query node

File: site/zh-CN/userGuide/load_balance.md [hunk 4]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
 	</tr>
 	<tr>
 		<td><code>dst_node_ids</code> (Optional)</td>
-		<td>ID(s) of the query node(s) you want to transfer segment(s) to. Milvus transfers segment(s) to other query nodes automatically if this parameter is left blank.</td>
+		<td>目标 segment 查询节点的 ID 。如果该参数为空，Milvus 会自动将 segment(s) 转移到其他查询节点。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:162]: 查询节点-->query node
~(S)~

File: site/zh-CN/userGuide/load_balance.md [hunk 5]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
 	</tr>
 	<tr>
 		<td><code>sealed_segment_ids</code> (Optional)</td>
-		<td>ID(s) of the segment(s) you want to transfer. Milvus transfers all sealed segment(s) in the source query node to other query nodes automatically if this parameter is left blank.</td>
+		<td>要转移的 segment ID 。如果该参数为空，Milvus 会自动将源查询节点中的所有封装 segment 传输到其他查询节点。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:166]: 封装 segment -->sealed segment
查询节点-->query node

File: site/zh-CN/userGuide/load_balance.md [hunk 6]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
-		<th>Description</th>
+		<th>参数</th>
+		<th>说明</th>
 	</tr>
 	</thead>
 	<tbody>
 	<tr>
 		<td><code>src_nodeID</code></td>
-		<td>ID of the query node you want to transfer segment(s) from.</td>
+		<td>源 segment 查询节点的 ID 。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:181]: 源 query node ID

File: site/zh-CN/userGuide/load_balance.md [hunk 7]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
 	</tr>
 	<tr>
 		<td><code>dst_nodeIDs</code> (Optional)</td>
-		<td>ID(s) of the query node(s) you want to transfer segment(s) to. Milvus transfers segment(s) to other query nodes automatically if this parameter is left blank.</td>
+		<td>目标 segment 查询节点的 ID 。如果该参数为空，Milvus 会自动将 segment(s) 转移到其他查询节点。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:185]: 目标 query node ID
~(s)~

File: site/zh-CN/userGuide/load_balance.md [hunk 8]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
 	</tr>
 	<tr>
 		<td><code>sealed_segmentIDs</code> (Optional)</td>
-		<td>ID(s) of the segment(s) you want to transfer. Milvus transfers all sealed segment(s) in the source query node to other query nodes automatically if this parameter is left blank.</td>
+		<td>要转移的 segment ID 。如果该参数为空，Milvus 会自动将源查询节点中的所有封装 segment 传输到其他查询节点。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:189]: 封装 segment -->sealed segment
查询节点-->query node

File: site/zh-CN/userGuide/load_balance.md [hunk 9]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
-		<th>Description</th>
+		<th>参数</th>
+		<th>说明</th>
 	</tr>
 	</thead>
 	<tbody>
 	<tr>
 		<td><code>SourceNodeID</code></td>
-		<td>ID of the query node you want to transfer segment(s) from.</td>
+		<td>源 segment 查询节点的 ID 。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:204]: 同上

File: site/zh-CN/userGuide/load_balance.md [hunk 10]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
 	</tr>
 	<tr>
 		<td><code>DestinationNodeID</code> (Optional)</td>
-		<td>ID(s) of the query node(s) you want to transfer segment(s) to. Milvus transfers segment(s) to other query nodes automatically if this parameter is left blank.</td>
+		<td>目标 segment 查询节点的 ID 。如果该参数为空，Milvus 会自动将 segment(s) 转移到其他查询节点。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:208]: 同上

File: site/zh-CN/userGuide/load_balance.md [hunk 11]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
 	</tr>
 	<tr>
 		<td><code>SegmentID</code> (Optional)</td>
-		<td>ID(s) of the segment(s) you want to transfer. Milvus transfers all sealed segment(s) in the source query node to other query nodes automatically if this parameter is left blank.</td>
+		<td>要转移的 segment ID 。如果该参数为空，Milvus 会自动将源查询节点中的所有封装 segment 传输到其他查询节点。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:212]: 同上

File: site/zh-CN/userGuide/load_balance.md [hunk 12]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
-		<th>Description</th>
+		<th>选项</th>
+		<th>说明</th>
 	</tr>
 	</thead>
 	<tbody>
 	<tr>
 		<td><code>-s</code></td>
-		<td>ID of the query node you want to transfer segment(s) from.</td>
+		<td>源 segment 查询节点的 ID 。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:227]: 同上

File: site/zh-CN/userGuide/load_balance.md [hunk 13]
```diff
@@ -148,104 +148,104 @@ load_balance -s 3 -d 4 -ss 431067441441538050
...
 	</tr>
 	<tr>
 		<td><code>-d</code> (Multiple)</td>
-		<td>ID(s) of the query node(s) you want to transfer segment(s) to.</td>
+		<td>目标 segment 查询节点的 ID 。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/load_balance.md:231]: 同上

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Hi @arthuo , We are so very delighted to have you in our community. And a huge thank for you generous contribution. 
I've made a few feedbacks based on your translation.
There is a quick note for you: you can have a quick scan of [Milvus components](https://milvus.io/cn/docs/v2.0.0/main_components.md) and [Glossary](https://milvus.io/cn/docs/v2.0.0/glossary.md). You will have a general knowledge of those words that we don't translate into Chinese.

**Comment by arthuo** [comment]
> Hi @arthuo , We are so very delighted to have you in our community. And a huge thank for you generous contribution. I've made a few feedbacks based on your translation. There is a quick note for you: you can have a quick scan of [Milvus components](https://milvus.io/cn/docs/v2.0.0/main_components.md) and [Glossary](https://milvus.io/cn/docs/v2.0.0/glossary.md). You will have a general knowledge of those words that we don't translate into Chinese.

Thank you very much for your documentation. I have made changes based on the feedback.

**Review by LocoRichard — APPROVED** [review]

**Comment by LocoRichard** [comment]
Excellent, merging the PR.

**Pull request merged**